ws/doc/p_dev_java_migrate.md

```diff
@@ -1,60 +0,0 @@
--   [Home](../../index.md)
```

Comment:
@vjwang  - please confirm that I can remove this 3.x-4.x migration content because I now link to the duplicate content in the Andriod topic